### DISCUSSION TITLE
Is it possible to use `torchmetrics` in GraphGym?

### DISCUSSION THREAD

**Discussion by do-lania** [pyg-team/pytorch_geometric#5723] [Q&A]
I am currently migrating my project to GraphGym and wondering if I can use my metrics written with `torchmetrics` there. I use both built-in and custom metrics in my project.

**Comment by rusty1s** [comment]
We currently use our own logic for implementing metrics but `torchmetrics` support would be super cool to add. Happy to work with you on this one together.
